### DISCUSSION TITLE
v4 translations are removed when server starts

### DISCUSSION THREAD

**Discussion by michelson** [fnando/i18n-js#621] [Q&A]
Hello, 
I've integrated the v4 in [our app](https://github.com/chaskiq/chaskiq), and we see that when the server starts (dev env), the JSON translations are emptied. 
How can we avoid this? have you seen this behavior?

thanks

**Comment by fnando** [comment]
Hey! Sorry, totally missed your question. Still facing this issue? If you do, can you please describe your current workflow, include the order of commands you're running?

**Reply by michelson** [reply]
HI, Right now I have this on the development.rb (using "i18n-js": "^4.0.0-alpha.4",)

```
  config.after_initialize do
    require "i18n-js/listen"
    I18nJS.listen(
      config_file: Rails.root.join("config/i18n.yml"),
      locales_dir: Rails.root.join("config/locales")
    )
  end
```
also, in initializer:

```
module I18nJS
  def self.translations
    ::I18n.backend.send(:init_translations)
    ::I18n.backend.send(:translations)
  end
end
```

This is working right now but don't know if this is proper way to do it. should I use a newer version?
